edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/billing/PurchaseStatus.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Billing
+{
+    public enum PurchaseStatus
+    {
+        NotFound,
```

Comment:
NP: Only 2 status? If there is a completed status I would have expect an "uncomplete" purchase or something :).